Cython/Build/Tests/TestInline.py

```diff
@@ -110,3 +111,11 @@ def test_numpy(self):
         a[0,0] = 10
         self.assertEqual(safe_type(a), 'numpy.ndarray[numpy.float64_t, ndim=2]')
         self.assertEqual(inline("return a[0,0]", a=a, **self.test_kwds), 10.0)
+
+    @unittest.skipIf(sys.version_info < (3, 6), "Requires Py3.6+")
```

Comment:
This isn't enough to keep the Python parser from seeing the new syntax. That makes it fail in older Python versions. Maybe we could use one of the other failing examples instead, e.g. one with a comment. Or use a separate test file.

Reply:
Yeah, I noticed that after submitting this, but couldn't think of a good quick fix.  I thought a string-based definition, plus an `eval`, might work, but the comment approach could be easier.  I'll try those later today.